packages/serverless-helpers/src/mocks/apiGateway.ts

```diff
@@ -38,7 +38,7 @@ export const getRequestContextMock = (
   ...args,
 });
 
-export const getRequestContextMockV2 = (
+export const getAPIGatewayEventRequestContextMockV2 = (
```

Comment:
```suggestion
export const getAPIGatewayV2EventRequestContextMock = (
```
Wouldn't it be a better naming?

Reply:
Yep, changed it